UPDATING.md

```diff
@@ -46,6 +46,15 @@ assists people when migrating to a new version.
 - [18976](https://github.com/apache/superset/pull/18976): When running the app in debug mode, the app will default to use `SimpleCache` for `FILTER_STATE_CACHE_CONFIG` and `EXPLORE_FORM_DATA_CACHE_CONFIG`. When running in non-debug mode, a cache backend will need to be defined, otherwise the application will fail to start. For installations using Redis or other caching backends, it is recommended to use the same backend for both cache configs.
```

Comment:
Found that this one is included on 1.5 branch, can you check?

Reply:
You're right, this one is included. I added it to the 1.5.0 changelog (will also add it to the 1.5 branch)